mirroroperator/registrymirror.py

```diff
@@ -254,64 +303,53 @@ def generate_daemon_set(self, daemon_set):
             )
         return daemon_set
 
-    def generate_service(self, service):
-        service.spec.type = "NodePort"
-        return service
-
     def generate_headless_service(self, service_headless):
         service_headless.spec.cluster_ip = 'None'
         service_headless.spec.type = "ClusterIP"
         return service_headless
 
-    def handle_proxy_credentials(self, env):
+    def get_nexus_credentials(self):
```

Comment:
rather than adding the credentials to the env, this method now just returns the credentials (or None if there are none)